tests/lib/rules/void-dom-elements-no-children.js

```diff
@@ -54,6 +54,10 @@ ruleTester.run('void-dom-elements-no-children', rule, {
     }, {
       code: 'React.createElement("img");'
     }, {
+      code: 'React.createElement();'
```

Comment:
Yup, a different rule that checks for required parameters to React methods would be good. Maybe it could even give a hint on the order and nature of the parameters without needing to go to the docs.